realm/realm-library/src/main/java/io/realm/internal/TableOrView.java

```diff
@@ -328,8 +328,6 @@
 
     TableView findAllDouble(long columnIndex, double value);
 
-    TableView findAllDate(long columnIndex, Date value);
```

Comment:
Yes, it has to be removed as Core doesn't support it. Again, core has never supported it but a clever hack made it possible to trick a non-checking core to perform the operation.